### DISCUSSION TITLE
[@types/react-copy-to-clipboard] 5.0.4

### DISCUSSION THREAD

**Discussion by nathanhannig** [DefinitelyTyped/DefinitelyTyped#63146] [Issues with a @types package]
Issue with React 18

'CopyToClipboard' cannot be used as a JSX component.
  Its instance type 'CopyToClipboard' is not a valid JSX element.
    The types returned by 'render()' are incompatible between these types.

[a comment by an automated account is not shown]

**Comment by nathanhannig** [comment]
A new version is available v5.1.0

**Reply by getellez** [reply]
but did it solve the problem?

**Reply by nathanhannig** [reply]
no
